lib/logstash/plugin_mixins/elasticsearch/api_configs.rb

```diff
@@ -11,15 +11,6 @@ module APIConfigs
         # Password to authenticate to a secure Elasticsearch cluster
         :password => { :validate => :password },
```

Comment:
Does this still work?

Reply:
I have ticket #5 check security . Will fix all security level clean up after having working CI for secure clusters.